api/server.js

```diff
@@ -0,0 +1,145 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const knexConfig = require("../knexfile.js");
+
+const db = knex(knexConfig.development);
+ 
+const server = express();
+server.use(express.json());
+server.use(helmet());
+
+
+// GET endpoint
+server.get('/api/zoos', (req, res) => {
+    db('zoos')
+    .then(zoos => {
+      res.status(200).json(zoos)
+    })
+    .catch(error => {
+      res.status(500).json(error)
+    })
+  });
+  
+  // POST endpoint
+  server.post('/api/zoos', (req, res) => {
+    const changes = req.body;
+  
+    db.insert(changes)
+    .into('zoos')
+    .then(ids => {
+      res.status(201).json(ids);
+    })
+    .catch(err => {
+      res.status(500).json({ error: "There was an error while saving the zoo to the database." });
+    })
+  })
+  
+  // GET by id endpoint
+  server.get('/api/zoos/:id', (req, res) => {
```

Comment:
you'll want to do the same kind of thing here - right now, if you type in an id that doesn't exist, it comes back with a 200 status and an empty object